app/views/hosts/_list.html.erb

```diff
@@ -8,6 +8,11 @@
         <th class="ca" width="5%"><%= _('Power') %></th>
       <% end %>
       <th width="25%"><%= sort :name, :as => _('Name') %></th>
+
+      <% Facets.registered_facets.values.each do |config| %>
+      <%= render "#{config.name.to_s.pluralize}/host/list_headers" if lookup_context.find_all("list_headers", "#{config.name.to_s.pluralize}/host/", true).any? %>
```

Comment:
We should add a helper for this. Or better two.
Something like:

`render_if_template_exists "#{config.name.to_s.pluralize}/host/list_headers"`

and
```ruby
def render_if_template_exists(template)
  render template if template_exists?(templete)
end

def template_exists?(templete)
  lookup_context.find_all("list_headers", "#{config.name.to_s.pluralize}/host/", true).any?
end
```